app/javascript/chat/chat.jsx

```diff
@@ -1794,6 +1794,11 @@ export default class Chat extends Component {
         ? 'sidecar-user'
         : `chat_channel_setting`;
 
+    const contentLink =
+      activeChannel.channel_type === 'direct'
```

Comment:
I'm getting a runtime error now here...

```javascript
Uncaught TypeError: activeChannel is null
```

Unless it's just something wrong locally for me.

Reply:
@benhalpern I am not getting this error locally. I will try to reproduce it.

Reply:
FWIW I didn't run into this problem locally either